### ISSUE TITLE
Various issues with 1.5.2

### ISSUE THREAD

**Issue by Kraker01** [9-9-9-9/Bit-Heroes-bot#30]
I'll write down in a single issue post:
1) "strange" characters like those at the beginning/end of sentences:
Select a function you want to launch:←[0m
←[33;1m** WARNING ** Please complete the above question first, otherwise bot will be hanged here!!!←[0m
10
←[94;1mYou selected function:←[0m
←[96;1m  Expedition←[0m
2) Expeditions: 
- Astamus portal isn't working (the bot move the cursors in the right spot, but isn't clicking it)
- auto-check is working (the bot correctly read the status of "auto" function) but doesn't enable it
3) World boss: when using this, i receive this error:
Key 'ig.user.raid.level' does not exists, default value '0' will be used (Not specified)
Key 'ig.user.raid.mode' does not exists, default value '0' will be used (Not specified)
Key 'ig.user.world-boss.level' does not exists, default value '0' will be used (Not specified)
←[31;1m** ERR ** Invalid World Boss level 0. Must in range 1 to 8←[0m
←[31;1m** ERR ** You have to do setting before using this function←[0m
←[31;1m** ERR ** Please launch script 'setting.bat' and follow instruction←[0m

This is my profile readonly.1.user-config.properties file:
Your setting:
  NORMAL mode of raid R1 (T4)
  world boss Orlag Clan (T3-T12)
Do you want to save the above setting into profile number 1 ?

**Comment by 9-9-9-9** [comment]
@Kraker01 

1. Its for coloring text, maybe your terminal isn't compatible with it. Please include `disable.colorize-terminal=true` to your `user-config.properties` to remove those strange characters. Really weird, as I know, Windows Command Prompt of Windows 10 does support it. I gonna check later on my W10 pc

2.1. How long was it stuck there?

2.2 How many time was the sentence `Detected Auto is not turned on, gonna reactive it soon` printed? Estimate at least 5 times (60 seconds).
```java
if (ImageUtil.isRedLikeColor(color)) {
    continousRed++;
    if (continousRed >= 2)
        info("Detected Auto is not turned on, gonna reactive it soon");
    if (continousRed >= 6) {
        moveCursor(point);
        sleep(100);
        mouseClick();
        hideCursor();
        info("Sent re-active");
        sleep(2_000);
    }
}
```

3. ~World Boss~
- ~Ya, first launch `setting.bat` app, do configure, save it by answer `Yes` when it asked you `Do you want to save the above setting into profile number 1 ?`~
- ~Next, launch `world boss` and specific profile:~
  - ~Answer  **`1`** when it asked `You want to do World Boss so you have to specific profile number first!
Select profile number`~
  - ~or simply by passing flag `--profile=1`: `java -jar BitHeroes.jar world-boss 9 --profile=1` (start world boss 9 times using profile number 1)~

**Comment by 9-9-9-9** [comment]
@Kraker01 ah sorry, the setting app was malfunctioned, gonna fix it in `1.5.3` soon

**Comment by 9-9-9-9** [comment]
@Kraker01 `3. World Boss` was fixed in [since release 1.5.3](https://github.com/9-9-9-9/Bit-Heroes-bot/releases)
by [this #31 pull request](https://github.com/9-9-9-9/Bit-Heroes-bot/pull/31)

**Comment by 9-9-9-9** [comment]
https://drive.google.com/file/d/1Do1Fc7JQRAWxdcyxfj7s_5vbIB_1HUqu/view?usp=sharing

![screenshot](https://drive.google.com/uc?export=view&id=1Do1Fc7JQRAWxdcyxfj7s_5vbIB_1HUqu)

^ thats how your terminal look like if the colorize feature works

**Comment by 9-9-9-9** [comment]
@Kraker01 `1. "strange" characters` was fixed [since release 1.5.5](https://github.com/9-9-9-9/Bit-Heroes-bot/releases)
by [this #33 pull request](https://github.com/9-9-9-9/Bit-Heroes-bot/pull/33)
